index.html

```diff
@@ -4,8 +4,29 @@
     <meta charset='utf-8'>
     <meta http-equiv='X-UA-Compatible' content='IE=edge'>
     <meta name='viewport' content='width=device-width, initial-scale=1'>
+
```

Comment:
Add a `<title>` for your webpage here! 

`<title>Your CodeSchool username here</title>`